src/mococrw/openssl_wrap.h

```diff
@@ -1339,6 +1341,19 @@ enum class ellipticCurveNid
     Ed25519 = NID_ED25519
 };
 
+enum class ellipticCurvePointConversionForm
+{
+    /* the point is encoded as z||x, where the octet z specifies
+     * which solution of the quadratic equation y is  */
```

Comment:
```suggestion
     * which solution of the quadratic equation x is  */
```
Maybe I'm misunderstanding things, but shouldn't this be `x` because that's what you use in `z||x` above?

Reply:
Hmm. Good question. I copied it from the openssl header (include/openssl/ec.h). Perhaps the solution of the elliptic curve equation is meant (y^2 = x^3 + ax + b). But I am also not sure about this.

Reply:
Hmh, I guess if you just copied it, it should be fine for now.